1224-user_administration/363-administrator_opens_the_add_role_page_part_i.feature

```diff
@@ -0,0 +1,14 @@
+    @363
+    Scenario Outline: Administrator opens the Add Role Page (Part I)
+        Given that the User is logged in as Administrator
+        And they are on the User Administration Page
+        When they click Add Role under the <Section>
+        Then they see the Add Role Page with fields:
+            | Authentication ID |
+        And they see Search Fields for first name and last name

```

Comment:
Er det egentlig nødvendig å ha et søkefelt for fornavn og et for etternavn? Synes det så ganske rart ut på demoen i dag. Er det ikke mer vanlig med kun et søkefelt? @terjehell